src/test/lib/parts_test.ts

```diff
@@ -102,16 +102,16 @@ suite('Parts', () => {
         part.setValue([1, 2, 3]);
         part.commit();
         assert.equal(stripExpressionMarkers(container.innerHTML), '123');
-        assert.strictEqual(container.firstChild, startNode);
-        assert.strictEqual(container.lastChild, endNode);
+        assert.strictEqual<Node>(container.firstChild!, startNode);
```

Comment:
this is a pattern i had to repeat a few times. its because `firstChild` is now a `ChildNode|null`, not just a `Node`. so `<T>(a: T, b: T)` doesn't work as `ChildNode|null` and `Node` aren't the same.